src/App.tsx

```diff
@@ -1,14 +1,99 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { TodoWithUser } from './types/TodoWithUser';
+import { FilterType } from './types/FilterType';
+import { ifInclude } from './components/IfInclude';
+
+async function loadTodos() {
+  const todosFromServer = await getTodos();
+
+  return todosFromServer;
+}
+
+async function loadUserForTodo(todo:Todo) {
+  const userFromServer = await getUser(todo.userId);
+
+  return userFromServer;
+}
 
 export const App: React.FC = () => {
+  const [isOpenedModal, setIsOpenedModal] = useState(false);
+  const [openedTodo, setOpenedTodo] = useState<TodoWithUser | null>(null);
+  const [isDataLoad, setIsDataLoaded] = useState(false);
+  const [isUserLoad, setIsUserLoaded] = useState(false);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todosServer, setTodosServer] = useState<Todo[]>([]);
+  const openModal = (todo:Todo) => {
+    setIsOpenedModal(true);
+    loadUserForTodo(todo).then(res => {
+      setOpenedTodo({
+        ...todo,
+        user: res,
+      });
+      setIsUserLoaded(true);
```

Comment:
not fixed 
<img width="508" alt="Screenshot 2023-04-08 at 18 28 58" src="https://user-images.githubusercontent.com/32853034/230732308-6519bc8e-1f7c-47fa-916d-2b2bf21a7190.png">